src/index.ts

```diff
@@ -1,3 +1,4 @@
+import 'newrelic';
```

Comment:
![magic](https://c.tenor.com/fA40mZBItgEAAAAM/shia-labeouf-magic.gif)

Meia dúzia de linha de código e tua aplicação já tem APM, log indexer, metrics... altas parada. É boa essa tal de New Relic, ein! 

Um salve para o @otaviocarvalho que faz as feature acontecer por lá 🙌

Reply:
Valeu a lembrança meus queridos! Esse agent de node é genial mesmo, respeito! 

Se não aparecer as entities, tags ou a busca não funcionar, pode ter certeza que é culpa minha 😅 

Enquanto isso, do outro lado da cortina:

<img src="https://media1.giphy.com/media/13FrpeVH09Zrb2/giphy.gif"/>

Reply:
@otaviocarvalho to confiando nesses teus commits ai feitos no papoeira comendo pastel 😆